Formula/t/transcrypt.rb

```diff
@@ -4,6 +4,7 @@ class Transcrypt < Formula
   url "https://github.com/elasticdog/transcrypt/archive/refs/tags/v2.3.1.tar.gz"
   sha256 "c5f5af35016474ffd1f8605be1eac2e2f17743737237065657e3759c8d8d1a66"
   license "MIT"
+  revision 1
```

Comment:
This will forced to make users update their app. Is that necessary?

Reply:
Not sure that I understand but you're suggesting not rebuilding the bottle?

If "brew install transcrypt" will skip xxd/vim without a revision, then indeed it's not necessary. Will it?

Reply:
Will not. But I thought this is not mendatory to the most of users and will be fine to be updated their app on the next release without a revision.

Reply:
Feel free to resolve, it is just an opinion.

Reply:
Shouldn't need a revision (nor a new bottle) as dependencies will be resolved at install time from formula's listed dependencies. Homebrew's JSON API will pick up new dependencies shortly after PR merge.

This is why we can have a single OS-independent `all` bottle even though the dependency trees are not the same:
https://github.com/Homebrew/homebrew-core/blob/4243613267696d6cae886c8303ce863ffce104d4/Formula/t/transcrypt.rb#L10

New `brew install transcrypt` will pick up updated dependencies. Users who want to potentially uninstall `vim` on an existing installation can run `brew reinstall transcrypt`.